app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express')
+const topicsRouter = require('./topics')
+
+const app = express()
+app.use(express.json())
+
+app.use('/api/topics', topicsRouter)
+
+app.use((req, res, next) => {
```

Comment:
Here as a first parameter you could add '*' to catch all invalid paths. 

An even better way is to use app.all(). (app.all('*', ()=>{}).